app/controllers/greetings_controller.rb

```diff
@@ -1,6 +1,6 @@
 class GreetingsController < ApplicationController
-    def index
-        @greeting = Greeting.all
-        render json: @greeting, status: :ok
-    end
+  def index
+    @greeting = Greeting.all
+    render json: @greeting, status: :ok
+  end
```

Comment:
### OPTIONAL
_I write it as an optional but this is not mentioned in the last review. So please fix it._
- [ ] Random greeting should come from the backend side. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md#:~:text=Create%20an%20API%20endpoint%20that%20selects%20a%20random%20greeting%20from%20your%20table%20(you%20will%20need%20a%20controller%20with%20an%20action%20an%20rails%20route).)